prowler/providers/aws/services/cloudfront/cloudfront_service.py

```diff
@@ -30,7 +30,25 @@ def _list_distributions(self, client, region) -> dict:
                         ):
                             distribution_id = item["Id"]
                             distribution_arn = item["ARN"]
-                            origins = item["Origins"]["Items"]
+                            origins = []
+                            ssl_protocols = []
+                            for origin in item.get("Origins", {}).get("Items", []):
+                                for protocol in (
+                                    origin.get("CustomOriginConfig", {})
+                                    .get("OriginSslProtocols", {})
+                                    .get("Items", [])
+                                ):
+                                    ssl_protocols.append(protocol)
+                                origins.append(
+                                    Origin(
+                                        id=origin["Id"],
+                                        domain_name=origin["DomainName"],
+                                        origin_protocol_policy=origin.get(
+                                            "CustomOriginConfig", {}
+                                        ).get("OriginProtocolPolicy", ""),
+                                        origin_ssl_protocols=ssl_protocols,
+                                    )
+                                )
```

Comment:
```suggestion
                                origins.append(
                                    Origin(
                                        id=origin["Id"],
                                        domain_name=origin["DomainName"],
                                        origin_protocol_policy=origin.get(
                                            "CustomOriginConfig", {}
                                        ).get("OriginProtocolPolicy", ""),
                                        origin_ssl_protocols=origin.get("CustomOriginConfig", {})
                                    .get("OriginSslProtocols", {})
                                    .get("Items", [])
                                    )
                                )
```
The protocols are already in a list :)